src/App.tsx

```diff
@@ -1,55 +1,154 @@
-import React from 'react';
+import React, { useCallback, useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
```

Comment:
<img width="909" alt="Screenshot 2023-09-05 at 17 31 28" src="https://github.com/mate-academy/react_autocomplete/assets/53228259/69d4e542-432a-42c5-b143-932a5c366d37">
you are using lodash but didn't install module in package.json
also try to write your own debounce function